src/components/FormStepSummary/SummaryEditGrid.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Utils as FormioUtils} from 'formiojs';
+import _ from 'lodash';
+
+import {getComponentLabel} from './utils';
+import {getBEMClassName} from '../../utils';
```

Comment:
we can use "absolute" imports (relative to the `src` directory) so we don't end up in path-hell:

```suggestion
import {getBEMClassName} from 'utils';
```